src/index.js

```diff
@@ -33,6 +34,8 @@ class RadioParser extends EventEmitter {
       this.setConfig(options);
     }
 
+    this._previousMetadata = {};
```

Comment:
BTW, we can make it an emtpy string here now, since we stringify them later anyway
```suggestion
    this._previousMetadata = '';
```